client/src/components/SingleVideo.js

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import { FaThumbsUp, FaThumbsDown } from "react-icons/fa";
+import "./SingleVideo.css";
+
+const SingleVideo = ({ video, handleDeleteVideoClick }) => {
+  const { title, url, rating, id } = video;
+  const [votes, setVotes] = useState(0); // can be 0, 1 or -1 => to add to rating when thumbsUp or ThumbsDown is clicked
+ 
+  const handleUpVote = () => {
+    setVotes(1);
+  };
+
+  const handleDownVote = () => {
+    setVotes(-1);
+  };
+
+  return (
+    <div className="video-card">
+      <h3>{title}</h3>
+      <div className="votes-container">
+        <div>
+          <FaThumbsUp
+            onClick={handleUpVote}
+            className={votes === 1 ? "voted thumb-icon" : "thumb-icon"}
+            size={30}
```

Comment:
It's fine setting the sizes individually like this, but as the app grows this would be a good candidate for becoming a variable eg `const iconSize = 30` that you can set in one place (towards the top of the `SingleVideo` function) and then use that `iconSize` variable to set both this and the thumbs down icon size. 

Then when you change your mind on icon sizes, it's one thing to change in one place.